authors/urls.py

```diff
@@ -13,12 +13,14 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
+from django.urls import path
```

Comment:
You can import both `path` and `include` from 
`from django.urls import path, include` and remove 
`from django.conf.urls import include, url` cause we will not use it

Reply:
Thanks @jeanjoe,Let me have i9t fixed